build.gradle.kts

```
@@ -33,7 +33,7 @@ buildscript {
 }
 
 plugins {
-  id("com.diffplug.spotless") version "5.17.0"
+  id("com.diffplug.spotless") version "5.1.0"
```

I assume you're after 5.17.1 here? Also, looks like 6.0.0 release this morning.
```suggestion
  id("com.diffplug.spotless") version "5.17.1"
```

---

Derp thanks. Will look at 6 in a follow-up